### DISCUSSION TITLE
Novice! Failed at the first hurdle!

### DISCUSSION THREAD

**Discussion by mayesp** [alphapapa/burly.el#71] [Q&A]
Trying out Burly, but seem to be doing something wrong. 

I have successfully enabled Burly - all the commands are available in my emacs session.

I have two frames open, with buffers in each.

I type "M-x burly-bookmark-frames RET", and Burly responds with

Save Burly bookmark: Burly:

The "Burly:" part is editable, but whatever I type, I get a "Wrong type argument: stringp, nil" error.

I should say I have never used emacs bookmarks before, and I have absolutely minimal understanding of elisp.

So I suspect this is a very basic usage error.

Thanks in advance -- Peter

**Comment by alphapapa** [comment]
Hello Peter,

I don't think this is a usage error, because the steps you've described are normal.

Please try to reproduce the problem in a clean Emacs configuration, e.g. using https://github.com/alphapapa/with-emacs.sh

If the problem still happens, enable `debug-on-error`, collect the backtrace when it happens, and file a bug report with detailed, complete steps to reproduce the problem.

Also, please make sure that you're using the latest versions of the relevant packages (using `with-emacs.sh` will help with that).

**Comment by mayesp** [comment]
Thanks for the response.

I tried in a clean configuration "with-emacs.sh -d foo -i burly", and it worked.

So I went back to my usual session, and now it works too, even though I don't think I changed anything! A mystery.

I came across burly the other day, as I wanted a way to save and restore a layout of emacs frames. Seems like burly does half of what I want - it opens the frames with their buffers as I want (dired, org-mode stuff etc). But it doesn't save the location on my screen. 

So I will keep burly loaded. But any ideas how to save the layout of windows, ideally on two monitors?

Thanks for your help -- Peter

**Reply by alphapapa** [reply]
> So I went back to my usual session, and now it works too, even though I don't think I changed anything! A mystery.

That probably indicates that one of the buffers' major modes has a bug in its bookmark support.  If the error happens again, the backtrace will probably point to the problem.

Burly doesn't do anything with regard to frame position or monitor placement.  It's not a bad idea, though, so feel free to file an issue with a feature request.  Alternatively, it might be better to make that suggestion with regard to my newer library, https://github.com/alphapapa/activities.el (which you'll likely want to use instead of Burly now).
